Library/Formula/vnu.rb

```diff
@@ -12,8 +10,7 @@ def install
   end
 
   test do
-    path = testpath/"index.html"
-    path.write <<-EOS
+    (testpath/"index.html").write <<-EOS.undent
```

Comment:
The `path` variable needs to be retained as it is used in the assertion below (the test isn't failing because `Formula` also defines an unrelated `path` method).